lerna.json

```diff
@@ -1,9 +1,11 @@
 {
   "lerna": "2.7.1",
   "packages": [
-    "components/**",
-    "tools/**",
-    "www"
+	  "components/**",
+	  "reactComponents/**",
```

Comment:
Can we rename them to just "vue" and "react" pretty please. Just incase we put utils or other things in there. Other than that im happy with this PR

Reply:
Sure, have changed these